src/tools/cgcreate.c

```diff
@@ -169,8 +174,8 @@ int main(int argc, char *argv[])
 		goto err;
 	}
 
-	/* this is false always for disable-systemd */
-	if (!ignore_default_systemd_delegate_slice)
+	/* this will always be false if WITH_SYSTEMD is not defined */
+	if (!create_scope && !ignore_default_systemd_delegate_slice)
```

Comment:
Both `create_scope` and `ignore_default_systemd_delegate_slice` are mutually exclusive, we need a check to check `T | T` premutation.

Reply:
I was thinking that as well.  Glad you brought it up.